### PR TITLE
Add logo

### PR DESCRIPTION
**Pull request by Teninq**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,6 +38,9 @@ Version](https://img.shields.io/github/r-package/v/insightsengineering/rlistings
 Issues](https://img.shields.io/github/issues-raw/insightsengineering/rlistings?color=red&label=open%20issues)](https://github.com/insightsengineering/rlistings/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc)
 <!-- end badges -->
 
+<img src="man/figures/rlistings-logo.svg" align="right" alt="" width="180">
```
Comment (Melkiades) [inline README.md:41]: I think it is better to put it on the right of the title as it is in `teal`:

`# Teal: Interactive Exploratory Data Analysis with Shiny Web-Applications <a href='https://insightsengineering.github.io/teal/'></a  >`

**Review by Melkiades — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Melkiades) [reply README.md:41]: what do you think @shajoezhu ?

Comment (shajoezhu) [reply README.md:41]: I was more thinking of similar to rtables actually

see 

```markdown
# rtables <a href='https://github.com/Roche/rtables'><img src="man/figures/rtables_logo_small.png" align="right" /></a>
```

https://raw.githubusercontent.com/Roche/rtables/main/README.md

Comment (shajoezhu) [reply README.md:41]: what do you reckon? @Melkiades @gmbecker

Comment (Melkiades) [reply README.md:41]: it is fine both imo

**Review by gmbecker — COMMENTED** [review]
Are there other NEST hex-logos that have a colored border like that? the one for `rtables` is white. If so, is the color being used in the border here used elsewhere within the NEST logo color pallete?

Other than that logo looks good

**Review by shajoezhu — APPROVED** [review]
lgtm! Thanks a lot @Teninq

File: README.md [hunk 2]
```diff
@@ -38,6 +37,8 @@ Version](https://img.shields.io/github/r-package/v/insightsengineering/rlistings
 Issues](https://img.shields.io/github/issues-raw/insightsengineering/rlistings?color=red&label=open%20issues)](https://github.com/insightsengineering/rlistings/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc)
 <!-- end badges -->
 
+
+
 ## Listings with R
```
Comment (Melkiades) [inline README.md:42]:
```suggestion
## Listings with R
```

**Comment by Melkiades** [comment]
@Teninq please adopt the branch [naming convention for staged.deps](https://github.com/openpharma/staged.dependencies#branch-naming-convention) (`Teninq:58_create_logo` should have been `58_create_logo@main`)

File: README.md [hunk 3]
```diff
@@ -1,8 +1,7 @@
 
 <!-- README.md is generated from README.Rmd. Please edit that file -->
 
-# rlistings
-
+# rlistings <a href='https://github.com/insightsengineering/rlistings'><img src="man/figures/rlistings.png" align="right" alt="" width="180"></a>
 <!-- start badges -->
 
 [![Check
```
Comment (Melkiades) [inline README.md:7]:
```suggestion

<!-- start badges -->
[![Check
```
This is just a way to restart checks as I cannot checkout your branch

**Review by Melkiades — APPROVED** [review]
Lgtm!! Thanks ;)

**Comment by Melkiades** [comment]
I thought this was the same problem from earlier @walkowif but it seems not. Is it some problem with the forking/branch naming?

**Comment by Melkiades** [comment]
> I thought this was the same problem from earlier @walkowif but it seems not. Is it some problem with the forking/branch naming?

Ok, it went in anyway. Nevermind ^^

**Pull request merged**